### DISCUSSION TITLE
Role-based restrictions such as no links for some roles

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-checklists#469] [Feature Requests]
I am building a sponsored guest post blog where there are three role levels with different permissions.

The first-level contributors can create posts but they can't add links to their posts.

It will be great if there is checklist that can handle it.

**Comment by devskywalk** [comment]
Yes, this would be a great feature. :) It is great for freemium model guest post blogs.

**Comment by rizaardiyanto1412** [comment]
This should be possible once we did this: https://github.com/publishpress/PublishPress-Checklists/issues/889
